Update Japanese localization based on the latest codebase

noritada opened the pull request:
This PR updates Japanese localization based on the latest codebase (b147f7ae8a5c8c3a1ec9981132b4c6447034963f).

Since many messages were marked for translation and context information was added to some messages, the number of untranslated messages in the latest codebase increased dramatically.

```shellsession
% # before reflecting changes in the latest code base
% LANG=C msgfmt --statistics src/locale/locales/ja/messages.po.
552 translated messages, 19 untranslated messages.

% # after reflecting changes in the latest code base
% LANG=C msgfmt --statistics src/locale/locales/ja/messages.po
542 translated messages, 344 untranslated messages.

% # after translation work
% LANG=C msgfmt --statistics src/locale/locales/ja/messages.po
744 translated messages, 142 untranslated messages.
```

In this PR, ja/messages.po was regenerated for the old codebase after the translation work was done to minimize diffs without making unnecessary changes, so the newly added messages are commented out (by running `yarn intl:build` on the old code base).
These messages should be auto-activated when the development team reflects the latest codebase into po files by `yarn intl:build`.

Dear @Hima-Zinn @tkusano @dolciss @oboenikui
I would appreciate your review and feedback on these text updates.

Many thanks!

oboenikui commented on src/locale/locales/ja/messages.po:
```diff
@@ -177,6 +269,22 @@ msgstr "および"
+
+#: src/view/com/modals/AddAppPasswords.tsx:133
+#~ msgid "App Password names can only contain letters, numbers, spaces, dashes, and underscores."
+#~ msgstr "アプリパスワードの名前には、文字、数字、スペース、ハイフン、アンダースコアのみが使用可能です。"
```
In Japanese, "文字" refers to a variety of characters, but the specifications seem to refer to English letters.

```suggestion
#~ msgstr "アプリパスワードの名前には、英数字、スペース、ハイフン、アンダースコアのみが使用可能です。"
```

Also need to fix https://github.com/bluesky-social/social-app/blob/e3a958b968f4a371dd76cf2cf8ade74b0c65aaa6/src/locale/locales/ja/messages.po#L488

noritada replied:
Thank you!

I made the translation to match the existing translation, but I was also concerned about it. You are right, I should change it.

Hima-Zinn replied:
Perhaps that is a remnant of the translation I did, so please change it to that one if you like.

noritada commented:
I have updated to make changes suggested.
Feel free to provide feedbacks if you have any other concerns.

Hima-Zinn commented:
I tried to edit the file and it created a fork and I edited it from there, but I don't know the GitHub specs and it didn't seem to merge there. What should I do?
https://github.com/bluesky-social/social-app/pull/2511

Hima-Zinn commented:
Thank you.

Please note that the header information is tentative and will need to be changed after various people have actually seen it, and that I have not incorporated @oboenikui's changes as I was not aware of them yet.

The code was too long, so if you can change the part I updated to something better, you should change it to that.

noritada commented:
Merged #2511 into this PR.

As for the header fields `POT-Creation-Date` and `MIME-Version`, I have reverted the changes back, as the other language teams have continued to use it without updating it from the beginning.

The changes I made based on @oboenikui's suggestions have indeed reverted to their previous state, so I have re-added them.

To clarify the changes, I have split into 3 commits.

- my first comimt: https://github.com/bluesky-social/social-app/pull/2509/commits/e3a958b968f4a371dd76cf2cf8ade74b0c65aaa6
- update based on @oboenikui's suggestions: https://github.com/bluesky-social/social-app/pull/2509/commits/b301242d15f63356a5eb238d73bb0bc4b473e8d8
- @Hima-Zinn's commit with minor changes by me: https://github.com/bluesky-social/social-app/pull/2509/commits/ca3948855497adc198ba281e15a5842cef02cdd8

tkusano reviewed (commented):
Looks good to me

dolciss reviewed (commented):
Thank you for the extensive changes.
I am sorry for the delay, but I have reviewed the contents and commented on some points, please confirm.

noritada commented:
@dolciss Thank you! I have made changes suggested.

英語との語順が異なるので、by 〜は悩みますよね…。
（私も後回しにしがちでした。）

dolciss commented on src/locale/locales/ja/messages.po:
```diff
@@ -472,23 +472,23 @@ msgstr "ビジネス"
 #: src/view/com/profile/ProfileSubpageHeader.tsx:157
 msgid "by —"
-msgstr ""
+msgstr "—により"
```
```suggestion
msgstr "作成者：—"
```

dolciss commented on src/locale/locales/ja/messages.po:
```diff
@@ -472,23 +472,23 @@ msgstr "ビジネス"
 #: src/view/com/profile/ProfileSubpageHeader.tsx:161
 msgid "by <0/>"
-msgstr ""
+msgstr "<0/>により"
```
```suggestion
msgstr "作成者：<0/>"
```

dolciss commented on src/locale/locales/ja/messages.po:
```diff
@@ -472,23 +472,23 @@ msgstr "ビジネス"
 #: src/view/com/profile/ProfileSubpageHeader.tsx:159
 msgid "by you"
-msgstr ""
+msgstr "あなたにより"
```
```suggestion
msgstr "作成者：あなた"
```

dolciss reviewed (commented):
Thank you for your quick response!
We would like to ask you to unify similar "by ~" messages as well.

noritada commented:
@dolciss Oops, fixed. Thank you very much!

noritada commented:
Since the release of 1.65 caused conflicts again, I have merged the latest changes into po to resolve the conflicts and translated 9 new messages.

New 9 translations are as follows!
https://github.com/bluesky-social/social-app/commit/06c9a201bcdba318844a73062a9780f6735570b1

(A message "We ran out of posts from your follows..." should be improved from the viewpoint of l10n. I have made a fix and will send a PR.)

Hima-Zinn commented:
I comment your commit in Japanese.

pfrazee commented:
@noritada Hey apologies for the long turn-around. Are we ready to merge?

noritada commented:
@pfrazee Sorry for my late reply. Yes, we are ready to merge!

pfrazee reviewed (approved):
Thanks everyone! <3 merging!
